### PR TITLE
Initial commit of version 0.1.0

### PR DESCRIPTION
**Pull request by blinktag**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Terraform Provider for Batch.sh
+
+* Website: [batch.sh](https://batch.sh)
+* Support Slack
```
Comment (dselans) [inline README.md]: Yank - add email `support@batch.sh` or point users to open a GH issue.

Comment (dselans) [reply README.md]: Also point to terraform docs on docs.batch.sh.

File: batch/filter.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/minio/pkg/wildcard"
+)
+
+type Filter struct {
+	Name   string
+	Values []string
+}
+
+func filterJSON(data []map[string]interface{}, filters []*Filter) ([]map[string]interface{}, diag.Diagnostics) {
```
Comment (dselans) [inline batch/filter.go:20]: Comment what is `[]map[string]interface{}` representing? What's the result?

**Review by dselans — APPROVED** [review]
Awesome!

File: batch/filter.go (hunk 2 above)
Comment (blinktag) [reply batch/filter.go:20]: Done!

File: README.md (hunk 1 above)
Comment (blinktag) [reply README.md]: Done!

**Pull request merged**